src/plugins/workspace/public/components/workspace_form/workspace_form.tsx

```diff
@@ -140,8 +140,9 @@ export const WorkspaceForm = (props: WorkspaceFormProps) => {
             errors={formErrors.selectedDataSources}
             onChange={setSelectedDataSources}
             savedObjects={savedObjects}
-            selectedDataSources={formData.selectedDataSources}
+            assignedDataSources={formData.selectedDataSources}
             data-test-subj={`workspaceForm-dataSourcePanel`}
+            isDashboardAdmin={isDashboardAdmin}
```

Comment:
`isDashboardAdmin` will always be true here, why we need to pass it from outside?

Reply:
thanks! it should be `true`